cb_medical_medication/views/medical_encounter_views.xml

```diff
@@ -16,7 +16,7 @@
                     class="oe_stat_button"
                     icon="fa-pills"
                     string="Add Medication"
-                    groups="cb_medical_medication.group_medical_medication_item_manager"
+                    groups="medical_base.group_medical_user"
```

Comment:
The group is not necessary, all users that can access encounters are already group_medical_user :smile: